lib/OpenQA/Worker.pm

```diff
@@ -14,108 +14,341 @@
 # with this program; if not, see <http://www.gnu.org/licenses/>.
 
 package OpenQA::Worker;
+use Mojo::Base -base;
 
-use 5.018;
-use strict;
-use warnings;
```

Comment:
And @Martchus made this package a `Mojo::Base` subclass, which automatically activates strictures. Both are fine and perlcritic treats them equally.

Reply:
Modules like [Mojo::Base](https://metacpan.org/pod/Mojo::Base#DESCRIPTION) and Moose (that act as object-systems) often activate strictures automatically. Because they basically "take over" the whole package when they are used and enforce a certain Perl dialect. Tools like perlcritic know this and treat `use Mojo::Base -base;` the same as `use strict; use warnings;`. That's also why they should be imported right after the package declaration, before other modules.

Reply:
So in conclusion this is no longer needed because it is now using `Mojo::Base` subclass.

Reply:
Agreed. Can we have an automatic check ensuring we are using the "right style" in all our code? E.g. no "use strict; use warnings" when there is only "Mojo::Base -base"?